app/models/foreman_fog_proxmox/proxmox_interfaces.rb

```diff
@@ -118,7 +118,7 @@ def dhcp?(nic_compute_attributes, v6 = false)
     def set_mac(nic_compute_attributes, mac, type)
       mac_attr_name = { 'qemu' => :macaddr, 'lxc' => :hwaddr }
       mac_key = mac_attr_name[type] || 'mac'
-      nic_compute_attributes[mac_key] = Net::Validations.normalize_mac(mac)
```

Comment:
This method does more like only downcase the MAC address. See: https://github.com/theforeman/foreman/blob/develop/lib/net/validations.rb#L122

would it be better to 1) run normalize_mac and afterwards do a upcase (if promox requires the mac address to be upcase?)

Reply:
Yes, created another PR for it